mlxtend/evaluate/accuracy.py

```diff
@@ -0,0 +1,57 @@
+# Sebastian Raschka 2014-2019
+# mlxtend Machine Learning Library Extensions
+#
+# A function for generating per class accuracy
+#
+# License: BSD 3 clause
+import numpy as np
+
+
+def _compute_metric(target_temp, predicted_temp):
+    return (target_temp == predicted_temp).sum() / float(target_temp.shape[0])
```

Comment:
This line,

    return (target_temp == predicted_temp).sum() / float(target_temp.shape[0])

could be even more simplified to

    return (target_temp == predicted_temp).mean()

I saw that you suggested the `normalize`. Since it is a simple thing, we could already add it here instead of a separate PR. When I understand this correctly, this would simply be

```python
def _compute_metric(target_temp, predicted_temp, normalize=True):
    if normalize:
        return (target_temp == predicted_temp).mean()
    else:
        return (target_temp == predicted_temp).sum()
```